Code/GraphMol/SLNParse/SLNParse.cpp

```diff
@@ -178,11 +178,11 @@ namespace RDKit {
     };
   } // end of SLNParse namespace
   
-  RWMol *SLNToMol(std::string sln,bool sanitize,int debugParse){
+  RWMol *SLNToMol(const std::string &sln,bool sanitize,int debugParse){
     // FIX: figure out how to reset lexer state
     yysln_debug = debugParse;
     // strip any leading/trailing whitespace:
-    boost::trim_if(sln,boost::is_any_of(" \t\r\n"));
+    //boost::trim_if(sln,boost::is_any_of(" \t\r\n"));
```

Comment:
This changes the behavior of the parser, right?
Any particular reason to do this aside from the fact that you have to since the function takes a const?